index.js

```diff
@@ -51,12 +51,7 @@ module.exports = (source, destination, options = {}) => {
 		}
 
 		if (files.length === 0) {
-			progressEmitter.emit('progress', {
-				totalFiles: 0,
-				percent: 1,
-				completedFiles: 0,
-				completedSize: 0
-			});
+			throw new CpyError(`Cannot copy \`${source}\`: no file found`);
```

Comment:
This doesn't sound good either.
```suggestion
			throw new CpyError(`Cannot copy \`${source}\`: the file doesn't exist`);
```

Same with the readme sentence.